src/main/java/net/geforcemods/securitycraft/screen/BlockPocketManagerScreen.java

```diff
@@ -24,6 +25,11 @@
 	private int size = 5;
 	private Button toggleButton;
 	private Button sizeButton;
+	private Button assembleButton;
+	private Button outlineButton;
+	private static final ItemStack blockPocketWall = new ItemStack(SCContent.BLOCK_POCKET_WALL.get());
+	private static final ItemStack chiseledCrystalQuartz = new ItemStack(SCContent.REINFORCED_CHISELED_CRYSTAL_QUARTZ.get());
+	private static final ItemStack crystalQuartzPillar = new ItemStack(SCContent.REINFORCED_CRYSTAL_QUARTZ_PILLAR.get());

```

Comment:
Names of static final fields (=constants) are written in upper snake case by convention. Example: `BLOCK_POCKET_WALL`. The names of the latter two variables also don't reflect the block that the ItemStack consists of. Please change.